tests/src/hipEnvVarDriver.cpp

```diff
@@ -102,7 +106,6 @@ int main() {
     // query the returned device pci bus number, check if match the database
     for (int i = 0; i < totalDeviceNum; i++) {
         setenv("HIP_VISIBLE_DEVICES", (char*)std::to_string(i).c_str(), 1);
-        setenv("CUDA_VISIBLE_DEVICES", (char*)std::to_string(i).c_str(), 1);
```

Comment:
@sunway513 This test is still failing on NVCC path. We would need to set CUDA_VISIBLE_DEVICES ( when __HIP_PLATFORM_NVCC__) , everywhere we are setting HIP_VISIBLE_DEVICES.